fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -890,7 +891,7 @@ fun Drawer(
                     modifier = modifier,
                     drawerState = drawerState,
                     drawerShape = drawerShape,
-                    drawerElevation = drawerElevation,
+                    drawerElevation = horizontalDrawerElevation,
```

Comment:
If HorizontalDrawer is intended to be used with just 0 dp elevation, maybe you can remove the drawerElevation parameter itself?